testing/idris/APKBUILD

```diff
@@ -0,0 +1,44 @@
+#-*-mode: Shell-script; coding: utf-8;-*-  
+# Maintainer: Mitch Tishmack <mitch.tishmack@gmail.com>
+pkgname=idris
+pkgver=1.0
+pkgrel=0
+pkgdesc="Idris - A Language with Dependent Types"
+url="http://www.idris-lang.org"
+arch="x86_64 armhf"
+license="bsd3"
+depends="libffi gmp ncurses libffi-dev gmp-dev"
```

Comment:
Looks like I was wrong its the other way around, idris needs gmp-dev when built with that as a feature but not libffi-dev to build ffi bindngs. I could probably leave out gmp as a feature if we wanted, but don't see much point.

```
/usr/lib/gcc/x86_64-alpine-linux-musl/6.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: cannot find -lgmp
collect2: error: ld returned 1 exit status
FAILURE: "gcc" ["-O2","-fwrapv","-fno-strict-overflow","-DHAS_PTHREAD","-DIDRIS_ENABLE_STATS","-I.","./testlib.o","/tmp/idris1669437588406334850.c","-L/usr/share/idris/rts/","-lidris_rts","-lgmp","-lpthread","-I/usr/share/idris/rts/","-lm","-I.","-I.","-I/usr/share/idris/libs/prelude","-I/usr/share/idris/libs/base","-o", "ffitest"]
# grep include  /tmp/idris1669437588406334850.c
#include "testlib.h"
#include "math.h"
#include "idris_rts.h"
#include "idris_bitstring.h"
#include "idris_stdfgn.h"
#include "idris_gmp.h"
#include "idris_opts.h"
#include "idris_rts.h"
#include "idris_stats.h"
#include <Windows.h>
```

Appears that its coming from the idris_gmp.h header.